docs/community/governance.md

```diff
@@ -58,11 +58,11 @@ activities. Core developers appear as organization members on the napari
 developers are expected to review code contributions while adhering to the
 [core developer guide](core-dev-guide). New core developers can be nominated
 by any existing core developer, and for details on that process see our core
-developer guide.
+developer guide. For a full list of core developers see our [About Us](team) page.
 
-### Steering Council
+### Steering council
```

Comment:
Both words should be capitalized here, yes? (since it is a proper noun)

Reply:
fixed